src/views/ApartmentApp/components/StudentApplication/components/BottomBar/index.js

```diff
@@ -102,41 +117,68 @@ const BottomBar = ({
               {dynamicContent.secondaryText}
             </Typography>
           </Grid>
-          {!applicationCardsOpen ? (
-            <Grid item xs={12} sm={4}>
-              <Button
-                variant="contained"
-                onClick={onShowApplication}
-                color="secondary"
-                fullWidth
-                disabled={applicationCardsOpen}
-              >
-                {dynamicContent.openCardsButtonLabel}
-              </Button>
-            </Grid>
-          ) : (
-            <Grid container item xs={12} sm={6} lg={4} spacing={2}>
+          <Grid container item xs={12} sm={7} lg={6} spacing={2}>
+            {!applicationCardsOpen && (
               <Grid item xs>
-                <SaveButton
-                  disabled={!canEditApplication || !unsavedChanges}
-                  buttonText={'Save & Continue'}
-                  status={saving}
-                  onClick={canEditApplication ? onSaveButtonClick : undefined}
-                />
-              </Grid>
-              <Grid item xs>
-                <SaveButton
-                  disabled={!canEditApplication || disableSubmit}
-                  buttonText={'Save & Submit'}
-                  status={submitStatus}
-                  onClick={canEditApplication ? onSubmitButtonClick : undefined}
+                <DynamicButton
+                  color={'secondary'}
+                  disabled={applicationCardsOpen}
+                  buttonText={dynamicContent.openCardsButtonLabel}
+                  onClick={onShowApplication}
                 />
               </Grid>
+            )}
+            <Grid item xs>
+              <DynamicButton
+                className={'delete-button'}
+                disabled={!canEditApplication || !applicationID}
+                buttonText={'Delete'}
+                startIcon={<DeleteIcon />}
+                status={deleting}
+                onClick={canEditApplication ? onDeleteButtonClick : undefined}
+              />
             </Grid>
-          )}
+            {applicationCardsOpen && (
+              <>
+                <Grid item xs>
+                  <DynamicButton
+                    color={'secondary'}
+                    disabled={!canEditApplication || !unsavedChanges}
+                    buttonText={'Save'}
+                    startIcon={<SaveIcon />}
+                    status={saving}
+                    onClick={canEditApplication ? onSaveButtonClick : undefined}
+                  />
+                </Grid>
+                <Grid item xs>
+                  <DynamicButton
+                    color={'primary'}
+                    disabled={!canEditApplication || disableSubmit}
+                    buttonText={'Submit'}
+                    startIcon={<PublishIcon />}
+                    status={submitStatus}
+                    onClick={canEditApplication ? onSubmitButtonClick : undefined}
+                  />
```

Comment:
This may be the intended functionality, but the Submit button is enabled a few seconds after you submit, even if you changed nothing. If it's not an easy fix, don't worry about it.

Reply:
Oh that is simple, i just have to add a "day submitted" check to the condition statement

Reply:
Nice!

Reply:
#1106